contributing/README.md

```diff
@@ -10,6 +10,6 @@ simply do the following:
 5. Create new Pull Request
 
 If you would like to see a bigger feature, or see `Pact` implemented in another
-language, please check out the [Pact specification](https://github.com/bethesque/pact-specification) and have a chat to one of us on the [pact-dev Google group](https://groups.google.com/forum/#!forum/pact-dev).
+language, please check out the [Pact specification](https://github.com/pact-foundation/pact-specification) and have a chat to one of us on the [pact-dev Google group](https://groups.google.com/forum/#!forum/pact-dev).
```

Comment:
We should probably remove the pact-dev google group and specify our slack at http://slack.pact.io instead